src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IConversionExpression.cs

```diff
@@ -0,0 +1,2979 @@
+        {
+            string source = @"
+using System;
+
+class C1
+{
+    delegate void DType();
+    void M1()
+    {
+        Delegate /*<bind>*/d2 =/*</bind>*/;
+    }
+
+    void M2()
+    {
+    }
+}
+";
+            string expectedOperationTree = @"
+IVariableDeclarationStatement (1 declarations) (OperationKind.VariableDeclarationStatement, IsInvalid) (Syntax: 'Delegate /* ... *</bind>*/;')
+  IVariableDeclaration (1 variables) (OperationKind.VariableDeclaration, IsInvalid) (Syntax: 'Delegate /* ... *</bind>*/;')
+    Variables: Local_1: System.Delegate d2
+    Initializer: IConversionExpression (ConversionKind.Invalid, Implicit) (OperationKind.ConversionExpression, Type: System.Delegate, IsInvalid) (Syntax: '')
+        IInvalidExpression (OperationKind.InvalidExpression, Type: ?, IsInvalid) (Syntax: '')
+";
+            var expectedDiagnostics = new DiagnosticDescription[] {
+                // CS1525: Invalid expression term ';'
+                //         Delegate /*<bind>*/d2 =/*</bind>*/;
+                Diagnostic(ErrorCode.ERR_InvalidExprTerm, ";").WithArguments(";").WithLocation(9, 43)
+            };
+
+            VerifyOperationTreeAndDiagnosticsForTest<VariableDeclaratorSyntax>(source, expectedOperationTree, expectedDiagnostics,
+                AdditionalOperationTreeVerifier: new ExpectedSymbolVerifier().Verify);
+        }
+
+        /// <summary>
+        /// This method is documenting the fact that there is no conversion expression here.
+        /// </summary>
+        [Fact]
+        public void ConversionExpression_Implicit_ReferenceMethodToDelegateConversion_NoConversion()
+        {
+            string source = @"
+class Program
+{
+    delegate void DType();
+    void Main()
+    {
+        DType /*<bind>*/d1 = M1/*</bind>*/;
+    }
+    void M1()
+    { }
+}
+";
+            string expectedOperationTree = @"
+IVariableDeclarationStatement (1 declarations) (OperationKind.VariableDeclarationStatement) (Syntax: 'DType /*<bi ... *</bind>*/;')
+  IVariableDeclaration (1 variables) (OperationKind.VariableDeclaration) (Syntax: 'DType /*<bi ... *</bind>*/;')
+    Variables: Local_1: Program.DType d1
+    Initializer: IMethodBindingExpression: void Program.M1() (OperationKind.MethodBindingExpression, Type: Program.DType) (Syntax: 'M1')
+        Instance Receiver: IInstanceReferenceExpression (InstanceReferenceKind.Implicit) (OperationKind.InstanceReferenceExpression, Type: Program) (Syntax: 'M1')
+";
+            var expectedDiagnostics = DiagnosticDescription.None;
+
+            VerifyOperationTreeAndDiagnosticsForTest<VariableDeclaratorSyntax>(source, expectedOperationTree, expectedDiagnostics);
+        }
+
+        [Fact]
+        public void ConversionExpression_Implicit_ReferenceMethodToDelegateConversion_InvalidIdentifier()
+        {
+            string source = @"
+class Program
+{
+    delegate void 
```

Comment:
I'm not sure what's going on here, as I can literally only see this comment on this page, and the actual delegate type on line 1421 is used: https://github.com/dotnet/roslyn/pull/20180/files#diff-03b0f60413e709897e7a71069c2cbc98R1421.